libs/sfm/bundler_common.cc

```diff
@@ -266,5 +266,50 @@ load_survey_from_file (std::string const& filename,
     in.close();
 }
 
+/* ---------------------- Feature undistortion -------------------- */
+
+namespace {
+    double distort_radius (double const r2, float const* params)
+    {
+        double coeff = 1.0 + r2 * params[0] + r2 * r2 * params[1];
+        return r2 * coeff * coeff;
+    }
+
+    double solve_undistorted_radius (double const r2, float const* params)
+    {
+        // Guess initial interval upper and lower bound
+        double lbound = r2, ubound = r2;
+        while (distort_radius(lbound, params) > r2)
+            lbound /= 1.05;
+        while (distort_radius(ubound, params) < r2 )
+            ubound *= 1.05;
+
+        // Perform a bisection until epsilon accuracy is reached
+        double const epsilon = 1e-10;
```

Comment:
You could use std::numeric_limits<double>::epsilon(), which is close to what you have, 1e-9.